snippets/ztp-hub-cluster-scale-test-specs.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: SNIPPET
+[IMPORTANT]
+====
+The following guidelines are based on internal lab benchmark testing only and do not represent a complete real-world host specification.
```

Comment:
Is "real-world" necessary here? I think you could remove it without taking away the meaning
```suggestion
The following guidelines are based on internal lab benchmark testing only and do not represent a complete host specification.
```